eevee/cli.py

```diff
@@ -43,14 +44,19 @@ def main():
 
         breakdown = True if args["--breakdown"] else False
         alias_yaml = args["--alias-yaml"]
+        groups_yaml = args["--groups-yaml"]
         return_output_as_dict = False
+        intent_aliases = None
         intent_groups = None
 
-        if not alias_yaml and breakdown:
+        if not groups_yaml and breakdown:
             raise ValueError("--breakdown requires, --alias-yaml along with it.")
 
         if alias_yaml:
-            intent_groups = parse_yaml(alias_yaml)
+            intent_aliases = parse_yaml(alias_yaml)
+
+        if groups_yaml:
```

Comment:
`groups_yaml` and `alias_yaml` are read by the same method, since they are both yaml's